app/.eslintrc.js

```diff
@@ -64,7 +64,7 @@ module.exports = {
         "@typescript-eslint/no-unused-vars": "error",
         // The usage of `any` defeats the purpose of typescript. Consider using `unknown` type instead instead.
         "@typescript-eslint/no-explicit-any": "error",
-      },
+\      },
```

Comment:
Syntax error on this line

Reply:
@coilysiren thank you! ran out of time yesterday, sorry for the churn on this one. Knock on wood, should be ready for re-review now 🙇‍♀️